packages/api/src/external/carequality/shared.ts

```diff
@@ -144,3 +145,22 @@ export function buildCqOrgName({
 
   return `${vendorName} - ${orgName}`;
 }
+
+export function getCqPatientContactType(
+  telecom: Telecom[] | null | undefined,
```

Comment:
nothing to do here but kinda silly we have a Telecom type for CQ and a Contact type for CW when they are really the same type if you dig into it. currently Telecom is a subset of Contact but I bet if we looked at the specs we would see that actually we are just ignoring values from Telecom and they are actually the same